src/cromshell/logs/command.py

```diff
@@ -0,0 +1,267 @@
+import logging
+
+import click
+import gcsfs
+from termcolor import colored
+
+from cromshell.metadata import command as metadata_command
+from cromshell.utilities import io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="logs")
+@click.argument("workflow_id")
+@click.option(
+    "-s",
+    "--status",
+    default="Failed",
+    help="Return a list with links to the logs with the indicated status. "
+    "Separate multiple keys by comma or use 'ALL' to print all logs. "
+    "Some standard Cromwell status options are 'ALL', 'Done', 'RetryableFailure', 'Running', and 'Failed'.",
+)
+@click.option(
+    "-p",
+    "--print-logs",
+    is_flag=True,
+    default=False,
+    help="Print the contents of the logs to stdout if true. "
+    "Note: This assumes GCS bucket logs with default permissions otherwise this may not work",
+)
+@click.option(
+    "-des",
+    "--dont-expand-subworkflows",
+    is_flag=True,
+    default=False,
+    help="Do not expand subworkflow info in metadata",
+)
+@click.pass_obj
+def main(
+    config,
+    workflow_id: str,
+    status: list,
+    dont_expand_subworkflows: bool,
+    print_logs: bool,
+):
+    """Get a subset of the workflow metadata."""
+
+    LOGGER.info("logs")
+
+    # If no keys were provided then set key_param to empty else
+    # strip trailing comma from keys and split keys by comma
+    status_param = (
+        ["ALL"]
+        if "ALL".lower() in status.lower()
+        else str(status).strip(",").split(",")
+    )
+
+    metadata_command.check_cromwell_server(config=config, workflow_id=workflow_id)
+
+    LOGGER.info("Status keys set to %s", status_param)
+
+    # To grab the logs we only need a subset of the metadata from the server
+    obtain_and_print_logs(
+        config=config,
+        metadata_param=[
+            "id",
+            "executionStatus",
+            "backendLogs",
+            "subWorkflowMetadata",
+            "subWorkflowId",
+        ],
+        status_params=status_param,
+        dont_expand_subworkflows=dont_expand_subworkflows,
+        cat_logs=print_logs,
+    )
+
+    return 0
+
+
+def obtain_and_print_logs(
```

Comment:
I would probably reorganize this somewhat there's a function that gets and returns the logs and then a separate one that prints them.